package.json

```diff
@@ -1,139 +1,107 @@
-    "console-read-write": "^0.1.1",
-    "date-format": "^3.0.0",
-    "eth-sig-util": "2.5.2",
-    "ethereum-cryptography": "^0.1.3",
-    "ethereumjs-common": "^1.5.2",
-    "ethereumjs-tx": "^2.1.2",
-    "ethereumjs-util": "^6.2.1",
-    "ethereumjs-wallet": "^1.0.1",
-    "ethers": "5.0.32",
-    "ethval": "^2.1.1",
-    "express": "^4.17.1",
-    "jsonrpc-lite": "^2.2.0",
-    "lodash": "^4.17.20",
-    "loglevel": "^1.7.0",
-    "minimist": "^1.2.5",
-    "nedb-async": "^0.1.3",
-    "ow": "^0.17.0",
-    "patch-package": "^6.4.6",
-    "semver": "^7.3.2",
-    "truffle-hdwallet-provider": "^1.0.17",
-    "web3": "1.2.6",
-    "web3-core": "1.2.6",
-    "web3-core-helpers": "1.2.6",
-    "web3-eth": "1.2.6",
-    "web3-eth-abi": "1.3.0",
-    "web3-eth-contract": "1.2.6",
-    "web3-utils": "1.2.6"
+    "@rsksmart/rif-relay-contracts": "github:rsksmart/rif-relay-contracts#PP-358/migrate-integration-tests"
   },
   "devDependencies": {
-    "@0x/sol-coverage": "4.0.8",
-    "@0x/sol-profiler": "4.0.8",
-    "@0x/sol-trace": "3.0.8",
-    "@0x/subproviders": "6.4.1",
-    "@openzeppelin/test-helpers": "0.5.10",
-    "@typescript-eslint/eslint-plugin": "^4.28.3",
-    "@typescript-eslint/parser": "^4.28.3",
-    "chai-as-promised": "7.1.1",
-    "cors": "2.8.5",
-    "eslint": "7.3.0",
-    "eslint-config-standard": "14.1.1",
-    "eslint-config-standard-with-typescript": "18.0.2",
-    "eslint-plugin-import": "2.21.2",
-    "eslint-plugin-node": "11.1.0",
-    "eslint-plugin-promise": "4.2.1",
-    "eslint-plugin-standard": "4.0.1",
-    "eth-gas-reporter": "^0.2.22",
-    "ganache-cli": "^6.12.1",
-    "ganache-core": "^2.13.1",
-    "husky": "^6.0.0",
-    "prettier": "2.3.0",
-    "rlp": "2.2.3",
-    "run-with-testrpc": "0.3.1",
-    "shx": "0.3.2",
-    "sinon": "9.0.2",
-    "sinon-chai": "3.5.0",
-    "solhint": "3.0.0",
-    "truffle": "5.1.67",
-    "ts-loader": "^9.2.3",
-    "ts-node": "8.6.2",
-    "typechain": "1.0.5",
-    "typechain-target-trufflehotfix": "0.0.4-alpha",
-    "typescript": "4.3.5",
-    "web3-provider-engine": "16.0.1",
-    "webpack": "^5.43.0",
-    "webpack-cli": "^4.7.2",
-    "webpack-node-externals": "^3.0.0"
+    "@commitlint/cli": "^17.4.0",
+    "@commitlint/config-conventional": "^17.4.0",
+    "@nomicfoundation/hardhat-toolbox": "^2.0.0",
+    "@nomiclabs/hardhat-ethers": "^2.2.1",
+    "@tsconfig/node16-strictest": "^1.0.4",
+    "@types/node": "^18.11.18",
+    "@types/sinon": "^10.0.13",
+    "@typescript-eslint/eslint-plugin": "^5.48.0",
+    "@typescript-eslint/parser": "^5.48.0",
+    "chai-as-promised": "^7.1.1",
+    "eslint-config-prettier": "^8.6.0",
+    "eslint-plugin-mocha": "^10.1.0",
+    "eslint-plugin-prettier": "^4.2.1",
+    "ethers": "^5.7.2",
+    "hardhat": "^2.12.5",
+    "husky": "^8.0.3",
+    "lint-staged": "^13.1.0",
+    "prettier": "^2.8.1"
   },
-  "types": "dist/index.d.ts"
+  "peerDependencies": {
+    "@openzeppelin/contracts": "^3.4.0"
+  }
```

Comment:
How come this is a peer dependency? this project is not going to be consumed is it?

Reply:
My fault, it should be probably included among the contracts peer dependencies, thank you for pointing it out.